src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
+        Storage.addFruit(fruit, quantity);
+    }
```

Comment:
The method `apply` in the `ReturnAction` class should check if the resulting balance after return operation is positive. According to the checklist, handling Purchase operation (and by extension, any operation that modifies the balance) should check the result balance before saving it in the Storage - it should be positive. Even though ReturnAction typically adds stock, it's a good practice to perform this check to ensure data integrity.